src/mbgl/util/grid_index.cpp

```diff
@@ -9,13 +9,13 @@ namespace mbgl {
 
 
 template <class T>
-GridIndex<T>::GridIndex(const float width_, const float height_, const int16_t cellSize_) :
-    width(width_),
-    height(height_),
-    xCellCount(std::ceil(width_ / cellSize_)),
-    yCellCount(std::ceil(height_ / cellSize_)),
-    xScale(xCellCount / width_),
-    yScale(yCellCount / height_)
+GridIndex<T>::GridIndex(const float width_, const float height_, const uint32_t cellSize_) :
+    width(std::fabs(width_)),
```

Comment:
do we expect negative `width_` (`height_`) ? should it be assert instead of `std::fabs`?

Reply:
Yes, assert would be better here. Fixed.